### PR TITLE
Remove unused forum stylesheet

### PR DESCRIPTION
**Pull request by tom93**
The forum was removed in commit 29bfab38 (Drop forem, replace with placeholder page to avoid dead links (#161), 2023-02-19).

### CODE REVIEW

**Comment by tom93** [comment]
Approve & merge after #126 due to merge conflicts.

**Comment by bagedevimo** [comment]
if you wanna rebase this i'll review it @tom93 ?

**Comment by tom93** [comment]
Rebased

**Review by bagedevimo — APPROVED** [review]

**Pull request merged**
